language/anope.de_DE.po

```diff
@@ -8262,8 +8321,9 @@ msgid ""
 "access entries on the channel."
 msgstr ""
 
+#, fuzzy
```

Comment:
There was a minor English change here to fix a typo. If this isn't fuzzy then you can remove this line.